cookbooks/fixed_slab_temperature/fixed_slab_temperature.prm

```diff
@@ -0,0 +1,154 @@
+# This is a cookbook demonstrating an application of fixing the temperature
+# in a compositional field. The model simulates 2D subduction of an arbitrary slab
+# generated using the Geodynamic WorldBuilder (Fraters et al. 2019), where the
+# temperature within the slab is fixed to the temperature it was initialized with.
+# The model runs for 15 Myr which allows for the slab to maintain its thermal state
+# while the mantle wedge convects and reaches a more dynamic thermal state.
+
+# Load additional library
+set Additional shared libraries            = ./libprescribed_temperature_in_field.so
+set World builder file                     = 2D_slab.wb
+set Use years in output instead of seconds = true
+set Output directory                       = output-longrun
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
+set Adiabatic surface temperature          = 600
+set Dimension                              = 2
+set End time                               = 50e6
+set Maximum time step                      = 2e6
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 2
```

Comment:
Is there a reason you only do 2 iterations? Wouldn't it make sense to do more to make sure the solution converges?

Reply:
This was just because it was slow on my mac, I'll rerun this on my desktop at NMT with a higher amount of iterations

Reply:
Will also update the figures to have bigger and easier to read colour bars